Library/Formula/phantomjs.rb

```diff
@@ -17,9 +17,13 @@ class Phantomjs < Formula
   depends_on "openssl"
 
   def install
-    inreplace "src/qt/preconfig.sh", "QT_CFG+=' -openssl -openssl-linked'",
-              "QT_CFG+=' -openssl -openssl-linked -I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}'"
-    system "./build.sh", "--confirm", "--jobs", ENV.make_jobs
+    if build.stable?
+      system "./build.sh", "--confirm", "--jobs", ENV.make_jobs,
+             "--qt-config", "-I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}"
+    else
+      inreplace "build.py", "/usr/local", HOMEBREW_PREFIX
```

Comment:
@Vitallium What would be ideal would be a README.md in https://github.com/Vitallium/phantomjs/tree/master/src/qt which noted for each submodule what the upstream branch was, what your fork's branch is (`phantomjs`?) and, if you're feeling extra generous, a small summary of the various patches.

> I wonder if we could extract the platform plugin and use the system installed version of Qt (not the QtWebkit tho).

As I mentioned on the PhantomJS CI issue: I think being able to use a system Qt (5) for as much as possible would make life much easier for both PhantomJS and downstream packagers. I wonder if it would be possible to build _just_ QtWebKit there.